aide_design/designs/civil_work/floc_classes/floc_chan.py

```diff
@@ -0,0 +1,58 @@
+from aide_design.shared.units import unit_registry as u
+from aide_render.builder_classes import DP, HP
+
+
+class Channel:
+    """This is a channel class. It's called by the flocculator class so
+    that the hierarchy of objects in Python is the same as in Fusion.
+
+    Attributes
+    ----------
+    These are the default values for a channel assembly. To overwrite, pass these
+    into the bod (Basis Of Design) variable into the constructor.
+
+    """
+
+    def __init__(self, num_chan, L_ent_tank_max, h_chan, L_sed, W_chan,
```

Comment:
I intentionally made them inputs so that everything would be dependent on a call from the floc class in order to avoid having the same variables (e.g. L_ent_tank_max) defined in two different places